.github/workflows/style-checks.yml

```diff
@@ -7,11 +7,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-    - uses: DoozyX/clang-format-lint-action@v0.13
+    - uses: DoozyX/clang-format-lint-action@v0.16.1
       with:
         source: 'library application python'
         extensions: 'h,cxx'
-        clangFormatVersion: 13
+        clangFormatVersion: 14 # Last Ubuntu LTS version (22.04)
```

Comment:
last ubuntu lts is 23.04

Reply:
Nop. 23.04 is not LTS.

Reply:
indeed

Reply:
maybe we should use 22.04 in the CI if EGL testing works with it. Looks like I got confused at some point.

Reply:
unrelated anyway